### PR TITLE
Added test_44782 for F-40E resource limit

### PR DESCRIPTION
**Pull request by aditipande96**
Signed-off-by: Aditi Pande <aditi.pande@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by aditipande96** [comment]
Collections are green:
====== 2352 tests collected in 20.40s ======

File: tests/csm/rest/test_csm_load.py [hunk 1]
```diff
@@ -409,3 +409,51 @@ def test_32547(self):
...
+        Verify max allowed CSM user limit
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.test_cfgs["test_44782"]
+        jmx_file = "CSM_create_max_manage_users.jmx"
+        self.log.info("Running jmx script: %s", jmx_file)
+
+        request_usage = test_cfg["request_usage"]
+        total_requests = test_cfg["total_requests"]
```
Comment (kachhwahadivya) [inline tests/csm/rest/test_csm_load.py:429]: This is total_users

Comment (aditipande96) [reply tests/csm/rest/test_csm_load.py]: done

File: tests/csm/rest/test_csm_load.py [hunk 2]
```diff
@@ -409,3 +409,51 @@ def test_32547(self):
...
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.test_cfgs["test_44782"]
+        jmx_file = "CSM_create_max_manage_users.jmx"
+        self.log.info("Running jmx script: %s", jmx_file)
+
+        request_usage = test_cfg["request_usage"]
+        total_users = test_cfg["total_users"]
+
+        loop = total_users // request_usage
```
Comment (kachhwahadivya) [inline tests/csm/rest/test_csm_load.py:431]: this wont work is request_usage is greater than total_users. Can u handle that scenario also.

Comment (aditipande96) [reply tests/csm/rest/test_csm_load.py:431]: request_usage is 30 and total_users is 97 so calculation comes to be 90 users in 3 loops and 7 users in last loop

**Review by kachhwahadivya — APPROVED** [review]

File: tests/csm/rest/test_csm_load.py [hunk 3]
```diff
@@ -409,3 +409,51 @@ def test_32547(self):
...
+        self.log.info("Run last batch of create csm users")
+        result = self.jmx_obj.run_verify_jmx(
+            jmx_file,
+            threads=req_last,
+            rampup=test_cfg["rampup"],
+            loop=1)
+        assert result, "Errors reported in the Jmeter execution"
+
+        #TODO: List users to verify if 100 users are present(99 csm+1 admin)
+        #Create one more user and check for 403 forbidden
```
[the remark this answers is not in the document]
Comment (aditipande96) [reply tests/csm/rest/test_csm_load.py]: done

**Review by RajeshPratapSInghChouhan — APPROVED** [review]

**Review by sonalk0209 — APPROVED** [review]
Approving for merge!!!

**Pull request merged**